pulsar-broker/src/main/java/org/apache/pulsar/PulsarStandalone.java

```diff
@@ -282,7 +282,7 @@ void start() throws Exception {
             workerConfig.setPulsarWebServiceUrl("http://127.0.0.1:" + config.getWebServicePort());
             if (!this.isNoStreamStorage()) {
                 // only set the state storage service url when state is enabled.
-                workerConfig.setStateStorageServiceUrl("bk://127.0.0.1:" + this.getStreamStoragePort());
+                // workerConfig.setStateStorageServiceUrl("bk://127.0.0.1:" + this.getStreamStoragePort());
```

Comment:
what is the reason of commenting this out?

Reply:
This is a stray change that I had in my workspace for debugging. Removed it